### PR TITLE
Open the secondary subject field when present

### PR DESCRIPTION
**Pull request by fofr**
### Context

When editing a secondary course with multiple subjects, the second subject field is closed, even if a subject is selected.

- Open the details element when the second subject is present
- Remove inline styles, replace with classes
- Tweak spacing
- Fix naming of test variable

(Ignore whitespace when reviewing)

https://trello.com/c/RyXdo7Ry/3512-open-the-secondary-subject-field-when-present

### CODE REVIEW

File: app/views/courses/subjects/_form_fields.html.erb [hunk 1]
```diff
@@ -16,24 +16,22 @@
...
-          </summary>
-          <div class="govuk-details__text" id="details-content" aria-hidden="true">
-            <p>Your first subject is the main one. It will appear first in the course title. It represents the bursary or scholarship available if applicable.</p>
+      <details class="govuk-details govuk-!-margin-top-6 govuk-!-margin-bottom-0" role="group" data-qa="course__subordinate_subject_details" <% if course.selected_subject_ids.second %>open<% end %>>
+        <summary class="govuk-details__summary" role="button">
+          <span class="govuk-details__summary-text">
+            Add a second subject (optional)
+          </span>
+        </summary>
+        <div class="govuk-details__text" id="details-content" aria-hidden="true">
```
Comment (aliuk2012) [inline app/views/courses/subjects/_form_fields.html.erb:25]: shouldn't the aria-hidden be set to "false" if the details element is open

File: app/views/courses/subjects/_form_fields.html.erb [hunk 2]
```diff
@@ -16,24 +16,22 @@
...
+          <p>Your first subject is the main one. It will appear first in the course title. It represents the bursary or scholarship available if applicable.</p>
```
Comment (aliuk2012) [inline app/views/courses/subjects/_form_fields.html.erb:26]: should this not be using `govuk-body` class?

Comment (fofr) [reply app/views/courses/subjects/_form_fields.html.erb:26]: Probably, I didn't change this line but will add

File: app/views/courses/subjects/_form_fields.html.erb (hunk 1 above)
Comment (fofr) [reply app/views/courses/subjects/_form_fields.html.erb:25]: Good catch

Comment (fofr) [reply app/views/courses/subjects/_form_fields.html.erb]: Going to remove this attribute as it isn't needed (it's not in the design system component)

**Pull request merged**